_tests/integration/ios_test.go

```diff
@@ -190,6 +216,26 @@ configs:
               - automatic_code_signing: api-key
               - cache_level: none
           - deploy-to-bitrise-io@%s: {}
+        build_for_testing:
+          steps:
+          - activate-ssh-key@%s: {}
+          - git-clone@%s: {}
+          - xcode-build-for-test@%s:
+              inputs:
+              - project_path: $BITRISE_PROJECT_PATH
+              - scheme: $BITRISE_SCHEME
+              - destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
+              - cache_level: none
+          - xcode-test-shard-calculation@%s:
+              inputs:
+              - shard_count: $TEST_SHARD_COUNT
+              - product_path: $BITRISE_XCTESTRUN_FILE_PATH
+          - deploy-to-bitrise-io@%s:
+              inputs:
+              - pipeline_intermediate_files: |-
+                  BITRISE_TEST_SHARDS_PATH
+                  BITRISE_TEST_BUNDLE_PATH
+                  BITRISE_XCTESTRUN_FILE_PATH
```

Comment:
You do not need to expose the `BITRISE_XCTESTRUN_FILE_PATH` env var because the `BITRISE_TEST_BUNDLE_PATH` is enough.